src/graph/two_sat.hpp

```diff
@@ -0,0 +1,91 @@
+#pragma once
+#include <bits/stdc++.h>
+using namespace std;
+
+struct two_sat {
+  public:
+    two_sat() : _n(0), scc(0) {}
+    two_sat(int n) : _n(n), _answer(n), scc(2 * n) {}
```

Comment:
```suggestion
struct TwoSAT {
  public:
    TwoSAT() : _n(0), scc(0) {}
    TwoSAT(int n) : _n(n), _answer(n), scc(2 * n) {}
```

[クラス名はライブラリ内ではPascalCase](https://github.com/YNUCPC/ynu-icpc-library/blob/main/CONTRIBUTING.md#%E5%91%BD%E5%90%8D%E8%A6%8F%E5%89%87)で統一しようと考えているので、このようなクラス名にするのはどうでしょうか？